px-app-nav.es6.js

```diff
@@ -56,14 +56,21 @@
        * and `label` defined. If an `icon` or `children` property is defined
        * on a subitem, those properties will be ignored.
        *
+       * The following optional properties can be used for any-level items:
```

Comment:
> ...can be used for any-level items

It only works on items with no children.

Reply:
Good catch, will update the comment